exporter/opentelemetry-exporter-prometheus/setup.cfg

```diff
@@ -41,8 +41,8 @@ package_dir=
 packages=find_namespace:
 install_requires =
     prometheus_client >= 0.5.0, < 1.0.0
-    opentelemetry-api >= 1.9.1
-    opentelemetry-sdk >= 1.9.1
+    opentelemetry-api >= 1.10
+    opentelemetry-sdk >= 1.10
```

Comment:
Should these be 1.10.0? I guess it doesn't really matter, other than for consistency